pkg/branding/brandedtext.go

```diff
@@ -0,0 +1,15 @@
+package branding
+
+const (
+	productBrandingNameRHACS = "Red Hat Advanced Cluster Security for Kubernetes"
+
+	productBrandingNameStackrox = "StackRox"
```

Comment:
I think both are clear enough to understand but yours is less verbose so that's an advantage. We do lose a little bit of consistency with the naming of the `ProductBrandingEnvName` variable and `getProductBrandingEnv()` function but I don't know if that matters.

On second thought this is already in the package branding so branding.productBrandingNameRHACS seems a bit repetitive. With that in mind I will change this and _also_ change branding.GetBrandedProductname() to branding.GetProductName()